src/rql/index.ts

```diff
@@ -129,6 +129,22 @@ export interface RQLBuilder {
    */
   excludes: (field: string, ...expressions: RQLString[]) => RQLBuilder;
 
+  /**
+   * @description skipCount() Skips the document counting step of a request to increase performance.
```

Comment:
I belief we refer to "records" rather the "documents" in other RQL operators 🤔

Reply:
i like that ALOT 🆗